tests/Integration/CliTest.hs

```diff
@@ -16,12 +18,27 @@ tests =
             world
                 |> run "elm-format" [ "--help" ]
                 |> expectExit 0
+        , world
+            |> run "elm-format-xxx" []
+            |> TestWorld.goldenStdout "usage instructions"
+                "tests/Integration/data/usage.stdout"
```

Comment:
I think you forgot to check this file in, or else forgot to change this path when you moved the file?